pkg/sarif.go

```diff
@@ -382,40 +349,55 @@ func createSARIFHeader(url, category, name, version, commit string, t time.Time)
 	}
 }
 
-func createSARIFRule(checkName, checkID, descURL, longDesc, shortDesc string,
+func generateRecommentation(remediation []string) string {
```

Comment:
nit: spelling of "recommen**d**ation"

Reply:
even worse.. this was supposed to be "remediation" :-). Fixed.